README.md

```diff
@@ -1060,8 +1056,6 @@ Bookmarks
 
 [Learn Shell Programming - Free Interactive Shell Programming Tutorial](http://www.learnshell.org/)
 
-[The Bash Guide](https://guide.bash.academy/)
```

Comment:
This link still works, not sure why there is a suggestion to remove it?

It could have been down temporarily, but it is active now.